modules/base/rendering/renderableplane.cpp

```diff
@@ -78,6 +79,31 @@ namespace {
         "aspect ratio of the content. Otherwise it will remain in the given size."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo ScaleByDistanceInfo = {
+        "ScaleByDistance",
+        "Scale By Distance",
+        "Decides whether the plane should automatically adjust in size to match the "
+        "distance to the camera. Otherwise it will remain in the given size."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ScaleRatioInfo = {
+        "ScaleRatio",
+        "Scale Ratio",
+        "The scale ratio for scaling a plane by distance to camera."
```

Comment:
Could you add another sentence here to add more information about permissible value ranges and what the ratio does?

Reply:
Yes please, I personally do not understand it. 

Is this the thing that decides how big the plane is allowed to become?  Maybe it can be changed to be a bit more intuitive?  Now it's unclear, at least to me, how this relates to the min and max size properties